imx/withdrawal.go

```diff
@@ -61,6 +61,10 @@ type TokenWithdrawal interface {
 	CompleteWithdrawal(ctx context.Context, c *Client, l1signer L1Signer, starkPublicKey string, overrides *bind.TransactOpts) (*types.Transaction, error)
 }
 
+// ERC20Withdrawal implements TokenWithdrawal. Used for withdrawal of ETh Tokens.
+type EthWithdrawal struct {
+}
```

Comment:
Do you mind explaining @suluvai why this is an empty struct?

Reply:
To create Eth token type no additional data required like `ERC20` or `ERC721` tokens. But to make the function name `CompleteWithdrawal` same for all token types. For that it needs to be implemented by an object so that we can use the same method name.